lvk/LVK.h

```diff
@@ -1002,6 +1002,7 @@ class ICommandBuffer {
 
   virtual void cmdFillBuffer(BufferHandle buffer, size_t bufferOffset, size_t size, uint32_t data) = 0;
   virtual void cmdUpdateBuffer(BufferHandle buffer, size_t bufferOffset, size_t size, const void* data) = 0;
+  virtual void cmdCopyBuffer(BufferHandle srcBuffer, size_t srcBufferOffset, BufferHandle dstBuffer, size_t dstBufferOffset, size_t size) = 0;
```

Comment:
The order or arguments should resemble `cmdCopyImage()` (and `vkCmdCopyBuffer()`):

```
virtual void cmdCopyBuffer(BufferHandle srcBuffer, BufferHandle dstBuffer, size_t srcOffset, size_t dstOffset, size_t size) = 0;
```